__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const request = require('supertest');
+const app = require('../app');
+
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+
+const {articleData, commentData, topicData, userData } = require('../db/data/test-data/index')
+
+beforeEach(()=>{
+    return seed({topicData, userData, articleData, commentData })
+})
+afterAll(() => {db.end()});
+
+describe('app', ()=>{
+    describe('api', ()=>{
+        describe('GET /api/topics', ()=>{
+            test('statuscode 200', ()=>{
+                return request(app).get('/api/topics').expect(200);
+            })
+            test('should return an array of topic objects', ()=>{
+                return request(app).get('/api/topics')
+                .then(({body})=>{
+                    const {topics} = body;
+                    expect(Array.isArray(topics)).toBe(true);
+                    topics.forEach((topic)=>{
+                        expect(typeof topic.slug).toBe('string');
+                        expect(typeof topic.description).toBe('string');
+                    }) 
+                })
+            })
```

Comment:
You were told in this ticket:

> As this is your first endpoint you may wish to also consider any general errors that could occur when making any type of request to your api. The errors that you identify should be fully tested for.

What could go wrong generally at this point that we can test for as a sad path? if you can identify what this is please test for it in this ticket as you refactor it or in the next ticket if you prefer. If you're unsure what I mean feel free to drop me a line.